lib/model/QubitAccessLog.php

```diff
@@ -48,4 +48,19 @@ public static function getPopularThisWeek(array $options = [])
 
         return $stmt->fetchAll();
     }
+
+    /**
+     * Delete old access_log records.
+     *
+     * @return int number of rows deleted
+     */
+    public static function expire(string $expiryDate): int
```

Comment:
Good thinking moving expiry logic to the data type's class!

Reply:
Thanks! :)